.github/workflows/release.yml

```diff
@@ -71,3 +71,10 @@ jobs:
           MAVEN_SETTINGS: ${{ secrets.MAVEN_SETTINGS }}
           build-cache: read-only
           run: etc/scripts/release.sh release_build
+      - name: Upload Staged Artifacts
```

Comment:
You can use this instead:

```yaml
artifact-name: io-helidon-build-tools-artifacts-${{ github.ref_name }}
artifact-path: target/nexus-staging/
```

See how the cli is archived in in validate.yml

Reply:
Done